.github/workflows/kserve-localmodel-agent-docker-publish.yml

```diff
@@ -97,7 +97,7 @@ jobs:
         with:
           platforms: linux/amd64,linux/arm64/v8,linux/ppc64le,linux/s390x
           context: .
-          file: Dockerfile
+          file: localmodel-agent.Dockerfile
```

Comment:
we will also need to onboard it to Konflux once we add this commit in the release branch.

Reply:
Yes konflux onboarding for ODH will be done once these changes are in the release branch. For downstream, the `dockerfile.konflux` files are already created